Gemfile

```diff
@@ -6,12 +6,12 @@ source 'http://gems.dev.mas.local'
 # development dependencies will be added by default to the :development group.
 gemspec
 
-ruby '2.3.3'
+ruby '2.5.3'
 
 gem 'bowndler'
 gem 'jquery-rails'
 gem 'rails', '~> 4.2.7'
-gem 'rubocop', '~> 0.49.1', require: false
+gem 'rubocop', require: false
```

Comment:
We have an history of issues caused by not setting rubocop to a specific version, rubocop being updated to latest in the testing build and then some rules failing due to changes brought by the new version.

I personally have been following this approach [from their readme](https://github.com/rubocop-hq/rubocop#installation) for the installation/bump:

> RuboCop's development is moving at a very rapid pace and there are often backward-incompatible changes between minor releases (since we haven't reached version 1.0 yet). To prevent an unwanted RuboCop update you might want to use a conservative version locking in your Gemfile:
> gem 'rubocop', '~> 0.63.1', require: false